### PR TITLE
Docs: Add pie chart docs

### PR DESCRIPTION
**Pull request by oddlittlebird**
- Added a Pie chart panel topic.
- Added a topic for legends content.
- Added a shared snippet for legends options that are common to multiple panels.

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-in-v7-5.md [hunk 1]
```diff
@@ -22,6 +22,10 @@ These features are included in the Grafana open source edition.
 
 Grafana 7.5 adds a beta version of the next-gen pie chart panel.
```
Comment (achatterjee-grafana) [inline docs/sources/whatsnew/whats-new-in-v7-5.md:23]:
```suggestion
Grafana 7.5 adds a beta version of the next-generation pie chart panel.
```

**Review by achatterjee-grafana — CHANGES_REQUESTED** [review]
Requesting one change.

File: docs/sources/panels/visualizations/pie-chart-panel.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## Display options
+
+Use the following options to refine your visualization:
+
+### Show
+
+Choose how much information to show.
+
+- **Calculate -** Reduces a series to a single value to be displayed.
```
Comment (oscarkilhed) [inline docs/sources/panels/visualizations/pie-chart-panel.md:31]: I think I got this wrong. It reduces each series to a single value per series.

Comment (oddlittlebird) [reply docs/sources/panels/visualizations/pie-chart-panel.md:31]: Fixed

File: docs/sources/panels/visualizations/panel-legend.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
++++
+title = "Panel legend options"
+weight = 50
++++
+
+# Legend
+
+You can use the legend adjust how a visualization displays series.
+
+This applies to the following visualizations:
```
[the remark this answers is not in the document]
Comment (achatterjee-grafana) [reply docs/sources/panels/visualizations/panel-legend.md:10]: Move this to line 8. There is no need for a separate line.

File: docs/sources/panels/visualizations/pie-chart-panel.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+title = "Pie chart panel"
+keywords = ["grafana", "pie chart"]
+weight = 850
++++
+# Pie chart v2 panel
+> **Note:** This documentation refers to a beta feature.
+
+The pie chart compares series from one or more queries.
+
+The pie chart displays reduced series or values in a series as they relate to each other in the form of slices of a pie. The arc length, area and central angle of a slice are all proportional to the slices value, as it relates to the sum of all values. This type of chart is best used when you want a quick comparison of a small set of values in an aesthetically pleasing form.
```
Comment (oscarkilhed) [inline docs/sources/panels/visualizations/pie-chart-panel.md:11]:
```suggestion
The pie chart displays reduced series, or values in a series, from one or more queries, as they relate to each other, in the form of slices of a pie. The arc length, area and central angle of a slice are all proportional to the slices value, as it relates to the sum of all values. This type of chart is best used when you want a quick comparison of a small set of values in an aesthetically pleasing form.
```

File: docs/sources/panels/visualizations/pie-chart-panel.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
++++
+title = "Pie chart panel"
+keywords = ["grafana", "pie chart"]
+weight = 850
++++
+# Pie chart v2 panel
+> **Note:** This documentation refers to a beta feature.
+
+The pie chart compares series from one or more queries.
```
Comment (oscarkilhed) [inline docs/sources/panels/visualizations/pie-chart-panel.md:9]:
```suggestion
```

File: docs/sources/panels/visualizations/panel-legend.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+- [Pie chart v2 panel]({{< relref "pie-chart-panel.md">}})
+- [Time series panel]({{< relref "time-series/_index.md" >}})
+
+## Toggle series
+
+Click on the series label in the legend to isolate the series in the visualization. All other series are hidden in the visualization. The data of the hidden series is still accessible.
```
Comment (achatterjee-grafana) [inline docs/sources/panels/visualizations/panel-legend.md:17]:
```suggestion
To toggle a series:
Click on the series label in the legend to isolate the series in the visualization. 
All other series are hidden in the visualization. The data of the hidden series is still accessible.
```

File: docs/sources/panels/visualizations/panel-legend.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+Cmd/Ctrl+click on the series label to hide the clicked series and remove the toggle.
```
Comment (achatterjee-grafana) [inline docs/sources/panels/visualizations/panel-legend.md:19]:
```suggestion
Use Cmd/Ctrl+click on the series label to hide the isolated series and remove the toggle.
```

File: docs/sources/panels/visualizations/pie-chart-panel.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+Select labels to display on the pie chart. You can select more than one.
+
+- **Name -** The series or field name.
+- **Percent -** The percentage of the whole.
+- **Value -** The raw numerical value.
+
+Labels are displayed in white over the body of the chart. You might need to select darker chart colors to make them more visible. Long names or numbers might be clipped.
+
+The example below shows a pie chart with **Name** and **Percent** labels displayed.
```
Comment (achatterjee-grafana) [inline docs/sources/panels/visualizations/pie-chart-panel.md:73]:
```suggestion
The following example shows a pie chart with **Name** and **Percent** labels displayed.
```

**Review by achatterjee-grafana — CHANGES_REQUESTED** [review]
Requesting a few changes.

File: docs/sources/panels/visualizations/panel-legend.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (oddlittlebird) [reply docs/sources/panels/visualizations/panel-legend.md:10]: I was kind of hoping @dprokop would expand on my anemic intro. :P

**Review by oscarkilhed — APPROVED** [review]
I’m happy with this :)

**Review by torkelo — APPROVED** [review]
Looks good!

**Review by achatterjee-grafana — APPROVED** [review]
LGTM

**Pull request merged**